components/03-modules/brief-info/mocks/brief-info.json

```diff
@@ -0,0 +1,64 @@
+{
+  "heading": {
+    "text": "Buy with us",
+    "class": ""
+  },
+  "items": [
+    {
+     "iconLabel": "Delivery truck",
+     "iconWrapperClass": "brief-info__icon-wrapper--delivery-truck",
```

Comment:
I don't like the way how it's done, b/c is not customizable without writing own styles.

This config file should be refactored, we will not get at least 1/3 of props from the backend.